.changeset/chatty-ducks-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": minor
+---
+
+Implement truncation for OnyxSelect options
```

Comment:
not sure if it _needs_ to be done like that, but I guess the majority of our changelog messages are in the syntax

```suggestion
feat(OnyxSelect): Implement truncation for OnyxSelectOptions
```